### PR TITLE
Hosted TodoMVC, Updated nix, added README.md

### PR DESCRIPTION
**Pull request by dmjio**
Changed nix files to build standalone, added link to hosted reflex todo mvc.

### CODE REVIEW

**Comment by dmjio** [comment]
my nix changes will probably break CI...

**Comment by dmjio** [comment]
Ah, seems like travis isn't setup.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# TodoMVC in Reflex - [Try It!](http://financial-controller-dolly-10433.bitballoon.com/)
```
Comment (ryantrinkle) [inline README.md]: I can add a cname for this, e.g. todomvc.reflex-frp.org; what do you think?

File: default.nix [hunk 2]
```diff
@@ -1,16 +1,3 @@
...
-  buildDepends = [
-    reflex
-    reflex-dom
-    file-embed
-  ];
-  license = null;
-}
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "ghcjs" }:
+	    nixpkgs.pkgs.haskell.packages.${compiler}.callPackage ./reflex-todomvc.nix
+	   { }
```
Comment (ryantrinkle) [inline default.nix:3]: Does this approach work with reflex-platform?

File: README.md (hunk 1 above)
Comment (dmjio) [reply README.md]: Yea, that'd be cool 😎

**Comment by dmjio** [comment]
@ryantrinkle I changed the url, will rebase / squash it.

File: default.nix (hunk 2 above)
Comment (dmjio) [reply default.nix:3]: I'll check on that ⌛

Comment (dmjio) [reply default.nix:3]: @ryantrinkle after updating `github.json` in `reflex-todomvc` (in `reflex-platform` repo)

``` json
{
  "owner": "dmjio",
  "repo": "reflex-todomvc",
  "rev": "4dc7079c35d7d8062643703bec9369585f64bee5", 
  "sha256": "18yfvrppbkq45zs6w5a5j9iyqdbskmx95zysvqv5gqjd7fvjmbl2"
}
```

Calling `nix-build` seems to have worked for me... also, what settings are you passing into `nix-hash` ? Is it simply `nix-hash --type sha256` ?

**Comment by Ericson2314** [comment]
Continued in https://github.com/reflex-frp/reflex-todomvc/pull/21

**Pull request closed without merging**